frontend/src/api/mutations/PlaceBet.ts

```diff
@@ -33,6 +36,7 @@ const putPlaceBet = (
     deposit: {
       id: Number(marketId),
       outcome: Number(args.outcomeId),
+      liquidity: LIQUDITY_PORTION,
```

Comment:
I'm assuming this is not configurable from the frontend ?

Reply:
Depends on what you mean. The frontend code itself sets this value to override the default in the contracts. There’s no UI exposing the option to override it.